### ISSUE TITLE
Package request: gtk-dlang

### ISSUE THREAD

**Issue by radare** [voidlinux/void-packages#1165]
http://gtkd.org/

**Comment by radare** [comment]
Requires dmd (but should build fine with ldc or ldc2 too acording tot he makefile) and gtk3-devel. Just download the zip and type `make`.

By default it is compiled without pic and as static archives. To compile the shared libs, use `make shared`, bear in mind that the makefile requires a little patch to find the available libphobos2 library.
```
-        $(if $(findstring "dmd","$(DC)"),$(eval LDFLAGS+=-defaultlib=:libphobos2.so))
+        $(if $(findstring "dmd","$(DC)"),$(eval LDFLAGS+=-defaultlib=:libphobos2-ldc.so))
```

`make install` seems to work fine

**Comment by Coding-Doctors** [comment]
This should be closed now since #8777 is merged

**Comment by maxice8** [comment]
@pullmoll please close this.
